agnpy/spectra/spectra.py

```diff
@@ -198,6 +200,42 @@ def plot(self, gamma=None, gamma_power=0, ax=None, **kwargs):
 
         return ax
 
+    def evaluate_time(self, time, energy_loss_function, subintervals_count=1):
+        unit_time_interval = time / subintervals_count
+
+        def gamma_recalculated_after_loss(gamma):
+            old_energy = (gamma * m_e * c ** 2).to("J")
```

Comment:
m_e c^2 is already defined in agnpy: https://github.com/cosimoNigro/agnpy/blob/75ab0b7fc25af8f8234b49cf6c33da0888baec7e/agnpy/utils/conversion.py#L6

Reply:
done